### PR TITLE
Clean up/bug fixes

### PR DESCRIPTION
**Pull request by jequeen**
- Fixed issue where when switching rotation 180 degrees facetracking controls would be inverted
- Updated version number on the settings screen
- Updated layouts to use `FragmentContainerView`

### CODE REVIEW

File: app/src/main/java/com/willowtree/vocable/BaseActivity.kt [hunk 1]
```diff
@@ -43,6 +46,61 @@ abstract class BaseActivity : AppCompatActivity() {
...
+    }
+
+    /**
+     * If the device rotates 180 degrees (portrait to portrait/landscape to landscape), the
+     * activity won't be destroyed and recreated. This means that the FaceTrackFragment will not
+     * reset its camera positioning. The only way to reset it currently is to create a new
+     * instance of the fragment and add it to the activity.
+     * @param tag The tag to use for the FaceTrackFragment, should be unique to the orientation
+     */
+    private fun resetFaceTrackFragment(tag: String) {
```
Comment (CarolineLaw) [inline app/src/main/java/com/willowtree/vocable/BaseActivity.kt]: nice 👍

File: app/src/main/res/layout-land/activity_settings.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
     android:orientation="vertical"
     android:theme="@style/SettingsTheme">
 
-    <fragment
+    <androidx.fragment.app.FragmentContainerView
```
Comment (CarolineLaw) [inline app/src/main/res/layout-land/activity_settings.xml]: how come we're using FragmentContainerView instead of fragment? just curious :]

Comment (nicholas-cook) [reply app/src/main/res/layout-land/activity_settings.xml]: This was related to the comments Matt Corbett made in #29 - `FragmentContainerView` is the new preferred method of adding fragments to an activity.

Comment (CarolineLaw) [reply app/src/main/res/layout-land/activity_settings.xml]: neat!

**Pull request merged**